src/Arcus.WebApi.Logging.Core/Correlation/HttpCorrelationInfoAccessor.cs

```diff
@@ -21,7 +20,10 @@ public class HttpCorrelationInfoAccessor : IHttpCorrelationInfoAccessor, ICorrel
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="contextAccessor"/> is <c>null</c>.</exception>
         public HttpCorrelationInfoAccessor(IHttpContextAccessor contextAccessor)
         {
-            Guard.NotNull(contextAccessor, nameof(contextAccessor));
+            if (contextAccessor is null)
+            {
+                throw new ArgumentNullException(paramName: nameof(contextAccessor));
```

Comment:
Reason why there's extensive argument names? In lots of cases (like here), it's unnecessary.

Reply:
I started adding these for clarity between the order of the arguments of `ArgumentException` and `ArgumentNullException`, since for some reason they are switched. But I stopped adding them when I didn't feel they added any clarity. I forgot to remove them in the first files I adapted.